src/stride.rs

```diff
@@ -167,6 +167,19 @@ macro_rules! stride_impl {
             pub fn len(&self) -> usize {
                 ((self.end - self.offset) / self.stride) as usize
             }
+
+            /// Returns the element of a stride at the given index, or None if the index is out of bounds.
+            #[inline]
+            pub fn get(&self, i: usize) -> Option<$elem> {
```

Comment:
Since the input is `&self` here the output must be a shared reference too `&A`. `$elem` is either `&A` or `&mut A`, so it's not correct for StrideMut.